test/graph/graph.builder.spec.js

```diff
@@ -31,7 +31,7 @@ describe("Graph Helper", () => {
         const sourceCoords = { x: 0, y: 0 };
         const targetCoords = { x: 0, y: 0 };
 
-        expect(linkHelper.buildLinkPathDefinition).toHaveBeenCalledWith(sourceCoords, targetCoords, "STRAIGHT");
+        expect(linkHelper.buildLinkPathDefinition).toHaveBeenCalledWith(sourceCoords, targetCoords, "STRAIGHT", []);
```

Comment:
No need to change here since we have a default assignment on the new `breakPoints` parameter.

Reply:
With my changes we do give `line.breakPoints` a default value twice.
1. in [graph.builder.js](https://github.com/danielcaldas/react-d3-graph/pull/378/files#diff-1801515e0b11550a614e14c0fa45110c76594b2bcb6dfcbdf971b501d09b5b0fR131).
2. in [link.helper.js](https://github.com/danielcaldas/react-d3-graph/pull/378/files#diff-a9bf4d32fe830d676e98ac37d85409bfe94f20b1f1cdaaaeac8f602a12e8033dR71).

Do you want me to remove the default assignment inside `graph.builder.js`, and change the line you mentioned to:
`...With(sourceCoords, targetCoords, "STRAIGHT", undefined);`?

Reply:
Now only buildLinkPathDefinition, has a default replacement for `link.breakPoints`.